helm/grafana/Chart.yaml

```diff
@@ -1,27 +1,20 @@
 apiVersion: v2
 name: grafana
-version: 1.1.0
+version: [[ .Version ]]
```

Comment:
I guess this goes along with changing the executor in the CI. Could we fix the error and keep the version here ?

Reply:
As I mentioned with Quentin, I have now put manually version: 2.0.0 there. Therefore we can keep the version here.